src/main/java/be/ugent/RDFWriter.java

```diff
@@ -100,7 +101,8 @@ public void setIfcReader(IfcSpfReader r) {
   }
 
   public void parseModel2Stream(OutputStream out) throws IOException {
-    ttlWriter = StreamRDFWriter.getWriterStream(out, RDFFormat.TURTLE_BLOCKS);
+	// CHANGED:  Jena  3.16.0    JO: 2020, added Context.emptyContext
+    ttlWriter = StreamRDFWriter.getWriterStream(out, RDFFormat.TURTLE_BLOCKS,Context.emptyContext);
```

Comment:
Should be `...RDFFormat.TURTLE_BLOCKS, Context.emptyContext);`
Note addition of space.